### ISSUE TITLE
importing segmentation masks 

### ISSUE THREAD

**Issue by pr4deepr** [Vizgen/vizgen-postprocessing#21]
Hi

  I am trying to import custom masks generated from cellpose. Currently they are tiff files, and there are around 220000 to 350000 cells depending on the tissue type. How do you recommend I generate the masks (geojson or hdf5)?

  I have been looking at creating polygons using 'utils' module from cellpose, but the process is super slow.  [Example functions](https://cellpose.readthedocs.io/en/latest/api.html#cellpose.utils.outlines_list_multi).

Cheers
Pradeep

**Comment by vdsukhov** [comment]
Hey @pr4deepr,

Personally, I have found the following approach to be very useful. If you already have a labeled image, you can use the `rasterio` package to convert it into the geojson format. This geojson file can then be used in the vpt pipeline. I use a script written by Pete Bankhead for QuPath to accomplish this.
https://gist.github.com/petebankhead/77782fd6d684e18efb2447980fdfbb90

**Comment by pr4deepr** [comment]
Thanks a lot @vdsukhov. 
Appreciate the link. Whats the speed like and have you encountered any issues?

**Comment by vdsukhov** [comment]
@pr4deepr In my experience, it works quite fast. I didn't measure the exact time, but I believe it took around 2 minutes. However, it did take some time to get it working with the vpt-pipeline. To troubleshoot, I created a single z-plane image of the dapi (pyramidal one) and opened it in QuPath. Then, I imported the geojson file in QuPath to verify the accuracy of the cell mask. Finally, I used it with the vpt.
